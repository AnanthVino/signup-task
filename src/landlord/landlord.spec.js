import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import LandLord from '.';
    
describe('LandLord', () => {
  it('should render correctly', () => {
    
    const wrapper = shallow(<LandLord />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})