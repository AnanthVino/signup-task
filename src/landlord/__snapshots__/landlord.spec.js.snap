// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandLord should render correctly 1`] = `
<div
  className="App"
>
  <nav
    className="navbar navbar-expand-lg navbar-dark text-light bg-primary"
  >
    <h3>
      Ajar Online
    </h3>
  </nav>
  <div
    className="d-flex justify-content-center"
  >
    <div
      className="d-flex form-box"
    >
      <div
        className="form-header text-light bg-primary text-center"
      >
        <h5>
          Signup as a landlord
        </h5>
      </div>
      <BrowserRouter>
        <Route
          component={[Function]}
          exact={true}
          path="/"
        />
        <Route
          component={[Function]}
          exact={true}
          path="/landlord"
        />
        <Route
          component={[Function]}
          exact={true}
          path="/landlord/stepone"
        />
        <Route
          component={[Function]}
          exact={true}
          path="/landlord/steptwo"
        />
      </BrowserRouter>
    </div>
  </div>
</div>
`;
